logicrunner/handle_calls.go

```diff
@@ -140,7 +163,7 @@ func (h *HandleCall) Present(ctx context.Context, f flow.Flow) error {
 		return errors.New("is not CallMethod message")
 	}
 
-	ctx, span := instracer.StartSpan(ctx, "LogicRunner.Execute")
+	ctx, span := instracer.StartSpan(ctx, "HandleCall.Present")
```

Comment:
and here you use proper message, i'm totally confused

Reply:
ok fixed